src/nl/hannahsten/texifyidea/run/pdfviewer/OkularViewer.kt

```diff
@@ -0,0 +1,43 @@
+package nl.hannahsten.texifyidea.run.pdfviewer
+
+import com.intellij.notification.Notification
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.project.Project
+
+/**
+ * Execute Okular commands.
+ *
+ * @author Abby Berkers
+ */
+object OkularViewer : SystemPdfViewer("Okular", "okular") {
+
+    /**
+     * Temporary variable to store the pdf file path.
+     */
+    private var pdfFilePath: String? = null
+
+    override val isFocusSupported: Boolean
+        get() = false
```

Comment:
This is correct, but isForwarSearchSupported should be true for all pdf viewers we currently have: Evince, Okular, Zathura, Skim, Sumatra

Reply:
Ah wait that is set in SystemPdfViewer, I see